docs/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Changed
+- Implement validation for embargoed flaws not be able to have public trackers (OSIDB-350)
+
+## [2.3.4] - 2022-12-15
+### Changed
```

Comment:
This still looks wrong. There is already version 2.3.4 in the changelog just below and I believe these changes you've tagged with this duplicate version are not released yet. :thinking:

Reply:
that is true they are not released yet, sorry, I messed up this file during a rebase, now I believe it is fixed.